opencda.py

```diff
@@ -35,6 +35,21 @@ def arg_parse():
                              'Set it to true only when you have installed the pytorch package and DI_engine RL '
                              'framework.')
 
+    parser.add_argument("--test_rl",
```

Comment:
adding too many argparse in the main file is usually not discouraged according to several coding style guidances. Here are my suggestions:
1. replace train_rl, test_rl, eval_rl with a single flag: rl_func, it should be a string, which allow 'train' 'test' or 'eval' only.
2. put the policy in your yaml file instead of here.
I will also try to remove apply_ml to the yaml file later after your branch is merged.